### PR TITLE
fix changelog white noise

### PR DESCRIPTION
**Pull request by gy95**
Signed-off-by: gy95 <guoyao17@huawei.com>

**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:
fix changelog 1.8 whitenoise, or the following PR will not pass the lint checks
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: CHANGELOG/CHANGELOG-1.8.md [hunk 1]
```diff
@@ -21,25 +21,25 @@ Download v1.8.0 in the [v1.8.0 release page](https://github.com/kubeedge/kubeedg
...
 CloudCore now supports Active-Active HA mode deployment, which provides better scalability support for large scale clusters.
-Cloud-Edge tunnel can also work with multiple CloudCore instances. 
+Cloud-Edge tunnel can also work with multiple CloudCore instances.
 CloudCore now can add the iptable rules for Cloud-Edge tunnel automatically.
 
 Refer to the links for more details.
 ([#1560](https://github.com/kubeedge/kubeedge/issues/1560), [#2999](https://github.com/kubeedge/kubeedge/pull/2999))
 
 
 **EdgeMesh Architecture Modification**
```
Comment (kevin-wangzefeng) [inline CHANGELOG/CHANGELOG-1.8.md:31]:
```suggestion
**EdgeMesh Architecture Modification**

```

File: CHANGELOG/CHANGELOG-1.8.md [hunk 2]
```diff
@@ -21,25 +21,25 @@ Download v1.8.0 in the [v1.8.0 release page](https://github.com/kubeedge/kubeedg
...
-at different locations. 
+EdgeMesh now has two parts: edgemesh-server and edgemesh-agent. The edgemesh-server requires a public IP address, when users use cross lan communication,
+it can act as a relay server in the LibP2P mode or assist the agent to establish p2p hole punching.
+The edgemesh-agent is used to proxy all application traffic of user nodes, acts as an agent for communication between pods
+at different locations.
 
 Refer to the links for more details.
 ([#19](https://github.com/kubeedge/edgemesh/pull/19))
 
 **EdgeMesh Cross LAN Communication**
```
Comment (kevin-wangzefeng) [inline CHANGELOG/CHANGELOG-1.8.md:40]:
```suggestion
**EdgeMesh Cross LAN Communication**

```

Comment (gy95) [reply CHANGELOG/CHANGELOG-1.8.md:41]: done

**Comment by gy95** [comment]
kindly ping @fisherxu  , PTAL, this PR need be merged as soon as possible, or will influence the new PRs.

**Review by fisherxu — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**